lang/c++/impl/DataFile.cc

```diff
@@ -54,7 +54,7 @@ const string AVRO_SNAPPY_CODEC = "snappy";
 #endif
 
 #ifdef ZSTD_CODEC_AVAILABLE
-const string AVRO_ZSTD_CODEC = "zstd";
+const string AVRO_ZSTD_CODEC = "zstandard";
```

Comment:
@zjw1111 Could you help split this critical fix into a separate PR if you don't have time to address all comments at this moment?